### ISSUE TITLE
SOUND BLASTER X-FI MB5 "Support device not found"

### ISSUE THREAD

**Issue by Bod4ik** [AlanFinotty/AAFOptimusDCHAudio-Generic#21]
SOUND BLASTER X-FI MB5 does not work with the drivers, giving a "support device not found" error. As far as I understand, the update Kb5023706 Windows 11 for some reason completely breaks the work of your driver. On the version of Windows before the update, everything works fine, but only in the original version, the alternative version still gives the same error. I tried to write the command "/RegisterSKU" in the terminal, but installing in KGA mode did not help. If possible, then I would install an older version of the alternative version of Sb X-Fi MB5. Can you upload an older version to a file sharing site?

![123](https://user-images.githubusercontent.com/57772693/226144716-4224d943-f736-4562-9e9d-eefd9cc691e9.png)

**Comment by senposage** [comment]
confirmed looks like Kb5023706
is breaking something

**Comment by Bod4ik** [comment]
> scrach that: uninstalling KBe5023706 doesn't fix it still says not disabled / not supported

Ok, here is a guide on how I managed to correctly run SB X-Fi MB5 on the latest version of Windows 11 (I think everything should work on Windows 10 22H2 too).
1. I uninstalled AAF drivers and X-Fi MB5
2. Removed leftovers via DriverStoreExplorer
3. Installed the latest AAF driver
4. Downloaded Alternative installer SB X-Fi MB5 and moved it to the root of drive C
5. In the active window with drive C, right-clicked and selected "Open in Terminal"
6. In the terminal window, write ".\"AAFSBXFiMB5Setup" /registerSKU" (without outer brackets)
7. Rebooted the PC and everything worked.
I hope this helps you, otherwise I myself have been dealing with the registerSKU command for a long time.
